app/index.html

```diff
@@ -25,6 +25,9 @@
 
     <meta property="og:image" content="/assets/og-image.png">
     <meta name="twitter:card" content="summary_large_image">
+
+    <link rel="preload" as="script" href="{{rootURL}}assets/vendor.js" />
+    <link rel="preload" as="script" href="{{rootURL}}assets/crates-io.js" />
```

Comment:
what is the performance gain for these?

I understand that they make sense at the top of the file when the file has a lot of server-rendered content, but as long as we have a fairly empty HTML file I'm wondering if they will actually have a positive effect.

Reply:
> what is the performance gain for these?
> 
> I understand that they make sense at the top of the file when the file has a lot of server-rendered content, but as long as we have a fairly empty HTML file I'm wondering if they will actually have a positive effect.

I'm extremely sorry if I am wrong; I thought browsers mark these scripts as high priority. Ref: https://nitropack.io/blog/post/link-rel-preload-explained

Reply:
<img width="1054" alt="Bildschirmfoto 2023-12-25 um 10 17 55" src="https://github.com/rust-lang/crates.io/assets/141300/164a2032-6613-4bc3-ad14-9f40c8431bf9">

(from https://www.webpagetest.org/result/231225_AiDcZ4_67C/1/details/#waterfall_view_step1)

if I interpret this waterfall diagram correctly, the JS and CSS files all start loading at the same time. since the preload hints in this PR come after the style links, they most likely wouldn't make a big difference compared to relying on the regular `script` tags for loading the files.

my understanding is that preload is only an advantage if a) the server-rendered page has a lot of HTML content in it and the script tags are at the bottom, or b) if e.g. a webfont is referenced in an CSS file but we could start loading the webfont already before loading and processing the CSS file.

Reply:
Hi @Turbo87, should I revert the preload-related change?

Reply:
I've interactive rebased on top of the `main` branch and dropped the preload commit

Reply:
Thanks

Reply:
Thanks